model/vm_pool.rb

```diff
@@ -5,7 +5,7 @@
 class VmPool < Sequel::Model
   one_to_one :strand, key: :id
   one_to_many :vms, key: :pool_id
-  one_to_many :idle_vms, key: :pool_id, class: Vm, conditions: {tainted_at: nil, display_state: "running"}
+  one_to_many :idle_vms, key: :pool_id, class: Vm, conditions: Sequel.&({tainted_at: nil}, Sequel.~(provisioned_at: nil))
```

Comment:
This is a Good Change and commit message.

Reply:
Thanks 🚀